src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -314,6 +311,36 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		}
 	}
 
+	private void updateFields() {
+		if (toolType.getSelectedItem() != null) {
+			blocksAffected.setEnabled(true);
```

Comment:
Changing order of these setEnabled calls doesn't change how the UI behaves, so I suggest that you, for the sake of consistency, order them all in the same way (preferably in order of declaration or in the same order as in the UI).

Reply:
I did changes here so it sets all to defaults first and then modifies them. Or did you mean something else?

Reply:
They are set in different orders for every case - a minor consistency issue

Reply:
Could you maybe fix OCD-related problem? :P Did you maybe test this?

Reply:
Can't right now, but the PR looks okay codewise.

Reply:
Do I have to change something?